### ISSUE TITLE
Add SVGR

### ISSUE THREAD

**Issue by marcomontalbano** [marcomontalbano/figma-export#34]
Add new component outputter based on [SVGR](https://react-svgr.com/).

**Comment by marcomontalbano** [comment]
Hi @berkaey, just to let you know that I released a new alpha version `2.0.0-alpha.2` that introduces TypeScript and the new outputter for React Components [@figma-export/output-components-as-svgr](https://www.npmjs.com/package/@figma-export/output-components-as-svgr).
The generated folder structure is a bit different from the one that you shared (https://github.com/marcomontalbano/figma-export/issues/25#issuecomment-576002827), but you should be able to get a similar result adjusting following configuration.

```js
module.exports = {
    commands: [
        ['components', {
            fileId: 'XXXX',
            onlyFromPages: ['Resources'],
            outputters: [
                require('@figma-export/output-components-as-svgr')({
                    output: './src/assets',
                    getDirname: (options) => options.dirname,
                    getFileExtension: () => '.js',
                    getSvgrConfig: (options) => {
                        const type = options.dirname.split('/').shift();
                        let replaceAttrValues = {};
                        if (type == 'icons' || type == 'logo') {
                            replaceAttrValues = {
                                '#6686F6': '{props.fill}',
                                '#1E1739': '{props.fill}',
                                '#000': '{props.fill}'
                            };
                        }

                        return {
                            native: true,
                            replaceAttrValues
                        };
                    }
                })
            ]
        }]
    ]
};
```

Let me know your opinion about it.
Thanks

**Comment by berkaey** [comment]
Thanks! I'll check and get back to you.
